gradle/libs.versions.toml

```diff
@@ -18,9 +18,9 @@
 #
 
 [versions]
-hadoop = "3.4.0"
+hadoop = "3.4.1"
```

Comment:
Can we do the version bumps in a separate PR?

Should actually be done by renovate ;)

Reply:
Sure, I can do in a separate PR. The reason why I included here it's because the transitive dependencies changed a bit.